packages/synapse-react-client/src/components/IconSvgButton.tsx

```diff
@@ -1,42 +1,71 @@
 import React from 'react'
-import { IconButton, Tooltip } from '@mui/material'
+import {
+  IconButton,
+  IconButtonProps,
+  SxProps,
+  Tooltip,
+  TooltipProps,
+} from '@mui/material'
 import IconSvg, { IconName } from './IconSvg/IconSvg'
 
-export type IconSvgButtonProps = {
+export type IconSvgButtonProps = Omit<IconButtonProps, 'children'> & {
   icon: IconName
-  tooltipText: string
-  onClick?: (e: React.MouseEvent) => void
-  disabled?: boolean
+  tooltipText?: string
   href?: string
-}
+  iconButtonSx?: SxProps
+  iconSx?: SxProps
+} & Pick<TooltipProps, 'placement'>
```

Comment:
Add tooltip placement option, and make tooltip text optional.  If no tooltip text, then do not wrap in a tooltip